example-user-library/README.md

```diff
@@ -13,54 +13,16 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -->
+# <img src="../logos/logo.svg" width="180">
+## A Tool for Complex and Scalable Data Access Policy Enforcement
 
-# Example Library
-
-The users, resources and policies to be used in the example are configured in the [configuration yaml](/example-library/src/main/resources/application-example.yaml).
-Each of these pre-population values are loaded into the appropriate service on service start-up, see the [run-example scripts](/deployment/local-jvm/example-runner).
-
-In particular, the example deals with the following users:
+# Example User Library
+Contains the ExampleUser and supporting code.
+Each of these users is defined by the ExampleUser which is a specialisation of the [Palisade-common](https://github.com/gchq/Palisade-common/README.md) User class.
+This will be used to create the following users:
 - User Alice has the role HR and completed the PAYROLL_TRAINING_COURSE
 - User Bob has the role ESTATES and not completed any training
 - User Eve has the role IT and not completed any training
```

Comment:
Might be worth expanding to give more of a blurb on likely types of queries each user would do to help link in the full scenario